src/components/AddToCartButton/AddToCartButton.tsx

```diff
@@ -10,21 +10,24 @@ it.
 import { FunctionComponent } from 'preact';
 
 import PlusIcon from '../../icons/plus.svg';
+import { useSensor } from '../../context';
 
 export interface AddToCartButtonProps {
   onClick: (e: any) => any;
 }
 export const AddToCartButton: FunctionComponent<AddToCartButtonProps> = ({
   onClick,
 }: AddToCartButtonProps) => {
+  const { screenSize } = useSensor();
+
   return (
-    <div className="ds-sdk-add-to-cart-button">
+    <div className={`ds-sdk-add-to-cart-button ${screenSize.desktop ? 'h-[100%] w-full' : 'h-[40px] w-[40px]'}`}>
```

Comment:
Why media queries couldn't be used here ?

Reply:
screenSize.desktop is a media query

Reply:
What I wanted to ask is  whether it can be done just in css ?

Reply:
the default tailwind config will serve our purpose though

Reply:
There is something wrong with nesting config
the media queries are not getting nested under .ds-widgets 
and hence the mobile classes are overriding the media queries classes

<img width="419" alt="image" src="https://github.com/user-attachments/assets/0c6a267a-66f3-4237-911d-b7c9de81aa67">